mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala

```diff
@@ -89,7 +89,8 @@ class StopWordsRemover @Since("1.5.0") (@Since("1.5.0") override val uid: String
   /**
    * Locale of the input for case insensitive matching. Ignored when [[caseSensitive]]
    * is true.
-   * Default: Locale.getDefault.toString
+   * Default: Locale.getDefault.toString, or Locale.US.toString if default locale isn't in
```

Comment:
.toString isn't actually helpful here.

Reply:
I think it just follows previous `Locale.getDefault.toString`. Getting rid of toString for both?

Reply:
I would get rid of it. You can replace Locale.US with the string value.